ui/vty/src/Ariadne/UI/Vty/Widget/Wallet.hs

```diff
@@ -291,6 +323,15 @@ updateFocusList = do
       , WidgetNameWalletSendRemove idx
       ]
 
+performRename :: WidgetEventM WalletWidgetState p ()
+performRename = do
```

Comment:
And this repeating stuff starts to look too boilerplate as well, but I don't think it should be generalized at the moment.